index.html

```diff
@@ -124,9 +123,10 @@ <h2>Introduction</h2>
           <li>Adapting the translation to the dubbing and subtitling specifications; ex. matching the actor’s lip movements in the case of dubs and considering reading speeds and shot changes.</li>
         </ul>
       </p>
-      <p>The result of creating a <a>dubbing script</a> can be useful as a starting point for creation of subtitles or closed captions in alternate languages;
-        This specification is designed to facilitate conversion to [[ttml-imsc1.1]] documents,
-        for example by permitting IMSC styling syntax to be carried in DAPT documents.
+      <p>The result of creating a <a>dubbing script</a> can be useful as a starting point for creation of subtitles or closed captions in alternate languages.
+        This specification is designed to facilitate conversion to [[ttml-imsc1.2]] documents,
```

Comment:
It could misinterpreted as the conversion not being easy to other versions of IMSC.

Reply:
Good point, will reword.